diptest/chat_test.go

```diff
@@ -17,22 +17,25 @@ func testChat(t *testing.T) {
 	sort.Sort(members)
 	chanName := strings.Join(members, ",")
 
+	WaitForEmptyQueue("game-asyncSendMsg")
+	WaitForEmptyQueue("game-sendMsgNotificationsToUsers")
+
 	t.Run("TestChatIsolationBetweenMembers", func(t *testing.T) {
-		startedGames[0].Follow("channels", "Links").Success().AssertEmpty("Properties")
-		startedGames[1].Follow("channels", "Links").Success().AssertEmpty("Properties")
-		startedGames[2].Follow("channels", "Links").Success().AssertEmpty("Properties")
+		startedGames[0].Follow("channels", "Links").Success().AssertLen(1, "Properties")
+		startedGames[1].Follow("channels", "Links").Success().AssertLen(1, "Properties")
+		startedGames[2].Follow("channels", "Links").Success().AssertLen(1, "Properties")
 
 		startedGameEnvs[1].GetRoute(game.IndexRoute).Success().
 			Follow("my-started-games", "Links").Success().
 			Find(startedGameID, []string{"Properties"}, []string{"Properties", "ID"}).
 			Find(startedGameNats[1], []string{"Properties", "Members"}, []string{"Nation"}).
-			AssertEq(float64(0), "UnreadMessages")
+			AssertEq(float64(1), "UnreadMessages")
```

Comment:
It would be nicer to verify what the message is. E.g. Check who it is from maybe?